setup.php

```diff
@@ -143,13 +143,24 @@ function cloneClicked() {
 <span class="text">
 
 <?php
+// Checks if the server's PHP version and the client's browser are compatible with OpenEMR:
+require_once(dirname(__FILE__) . "/common/compatibility/checker.php");
+
+$compatCheck = new Checker();
+$potentialWarnings = $compatCheck->getCompatibility();
+if (!empty($potentialWarnings)) {
+  echo "<script>alert('" . $potentialWarnings . "');</script>";
```

Comment:
run this through addslashes to escape:
``````
cho "<script>alert('" . addslashes($potentialWarnings) . "');</script>";